README.md

```diff
@@ -1,37 +1,95 @@
-# noms
+# 🍱 noms
+[![Super Linter](https://github.com/googleinterns/noms/workflows/Super%20Linter/badge.svg)](https://github.com/googleinterns/noms/actions)
 
-A full-stack web application that connects students and college organizations by posting about free food events on campus.
 
-## Context
+>A full-stack web application connecting students and college organizations to battle food insecurity and food waste.
 
-In the US, there are 4,360 degree granting institutions in the United States; including 2-year and 4-year colleges. Through all of these colleges, there are 19.9 million students who attend and specifically at 4-year colleges, 85% are awarded financial aid. Averaging the amount of college students and institutions, there are around 4500 students per campus; college campuses host a large number of organizations, which frequently have catered events, ranging from quick club meetings to multi-day academic conferences and hackathons. It is difficult for these organizations to accurately predict how many attendees their events will have, and they may err on the side of ordering too much food. When these events end, the leftover food needs to be consumed quickly or risk being thrown away. This kind of potential food waste is, environmentally speaking, the worst kind there is: food that has completed the entire cycle of growing, transportation, storage, and preparation—just to end up in the bin. 
+## Overview
+- [Description](#Description)
+- [Motivation](#Context)
+- [Technologies](#Technologies)
+- [Demos](#Demos)
+  - [Find your college](#find-your-college)
+  - [Find food at your college](#find-food-at-your-college)
+  - [Make a new post](#make-a-new-post)
+  - [Sign up for email notifications](#sign-up-for-email-notifications)
+  - [Receive email notifications](#receive-email-notifications)
+- [Usage](#Usage)
+- [Authors](#Authors)
+- [Contributing](#Contributing)
+- [License](#License)
+- [Disclaimer](#Disclaimer)
 
-Combine this issue with the reality that at some colleges, up to half of all students regularly experience food insecurity, and getting leftover food into the right hands (and stomachs) becomes not just an environmental necessity, but a humanitarian one as well. However, in attempting to address these issues, college organizations may find they lack the infrastructure to quickly notify and distribute their leftover food to the students who need it most, despite the fact that the students are otherwise ideally located to receive the food.
+## Description
```

Comment:
For internal users/reviewers, should we create a separate section with go/links that would be helpful like the app, feedback form, bug form, and our design docs?

Reply:
I'm not sure if go/links are allowed to be public? Like I understand they're not accessible to anyone outside of the corp network but I'm not sure if they're allowed to be mentioned, might have to ask Sanya or Natalie

But if they are, I'd be down to add a section!

Reply:
If not go/links, the links itself would be nice!

Reply:
Actually, upon thinking about it more, I think that the design doc itself should be the main hub for these links for a couple reasons:

- Anyone who can see our design doc is already corp, so they'll be able to access all of the links (I imagine random people coming across this repo might be miffed if a whole section of links doesn't work for them)
- Related to the above point, I don't think that the links in the readme would age well; after the internship is over, some of those links won't be relevant anymore (such as the feedback/bug forms, as we won't be able to access responses), and removing them from the readme would require 3 reviewers or removing branch protections
- We can use go/links without issue in the design doc
- The design doc is the main hub of documentation on our project anyway (it branches out to our feature docs)

Lemme know if any of this points seem unreasonable or iffy!